frontend/public/locales/en/translation.json

```diff
@@ -375,5 +375,49 @@
     "videos": "videos",
     "entityCandidate": "candidate",
     "entityVideo": "video"
+  },
+  "presidentielle2022": {
+    "dialogs": {
+      "tutorial": {
+        "title1": "Welcome on Tournesol \uD83C\uDF3B",
+        "message1": {
+          "p10": "In order to familiarize yourself with the comparison system, small messages like this one will be displayed.",
+          "p20": "Let's start with a first comparison between two candidates.",
+          "p30": "Move the blue handle below the candidates, then submit to select the person who should be president.",
+          "p40": "The more the handle is in the center, the more the candidates are considered similar."
+        },
+        "title2": "Congratulations !",
+        "message2": {
+          "p10": "Let's continue with other candidates.",
+          "p20": "Note that you can manually choose the candidates by using the drop down menu located just above the candidates' photos."
```

Comment:
Should "menu" be plural?

Reply:
I think so.

Fixed by https://github.com/tournesol-app/tournesol/pull/705/commits/f963a07da5157a207f4a7579ecd252cbed9fc5b3.